pootle/settings/91-travis.conf

```diff
@@ -16,28 +16,31 @@ if os.environ.get("TRAVIS"):
     DATABASE_BACKEND = os.environ.get("DB")
     DATABASES = {
         'default': {
-            'ENGINE': 'django.db.backends.sqlite3',
-            'NAME': working_path('dbs/zing_travis.db'),
-            'USER': '',
+            'ENGINE': 'django.db.backends.mysql',
+            'NAME': "zing",
+            'USER': 'travis',
             'PASSWORD': '',
             'HOST': '',
             'PORT': '',
             'ATOMIC_REQUESTS': True,
+            'CHARSET':'utf8mb4',
```

Comment:
Do we need to override CHARSET and COLLATION now that they are in `90-local.conf.template` file? (90-local.conf file will be loaded before 91-travis.conf)

Reply:
But is Travis loading `90-local.conf.template` on the build environment? It seems to me that It should not run it since it has the `.template` extension

Reply:
Any input on this Igor? As I stated above It's not clear to me that travis will use the configuration in `90-local.conf.template`